cmd/scheduler/main.go

```diff
@@ -24,14 +24,42 @@ const (
 	retryExpRate  = 1.5
 )
 
+type ManagerConfig struct {
+	// ExcludeVersions is a list of regexp expressions that will ignore a
+	// specific version of a package if it is matched by any of the expressions.
```

Comment:
The wording is a little ambiguous - I thought it might apply the rule to only certain specified packages.
Suggest rewording, e.g. "Versions of any package whose version string matches a regexp in this list will be ignored"

Reply:
Updated. PTAL

Reply:
Awesome, looks good!